test/plugin_functional/test_suites/custom_visualizations/self_changing_vis.js

```diff
@@ -45,20 +41,9 @@ export default function({ getService, getPageObjects }) {
       const isApplyEnabled = await PageObjects.visEditor.isApplyEnabled();
       expect(isApplyEnabled).to.be(true);
       await PageObjects.visEditor.clickGo();
+      await renderable.waitForRender();
       const counter = await getCounterValue();
       expect(counter).to.be('10');
     });
-
-    it('should allow changing params from within the vis', async () => {
```

Comment:
leave this test skiped (don't remove it)

Reply:
done